assets/global.js

```diff
@@ -206,6 +205,103 @@ class QuantityInput extends HTMLElement {
 
 customElements.define('quantity-input', QuantityInput);
 
+class PricePerItem extends HTMLElement {
```

Comment:
Can you move this to its own file? We wont need it for every page. This way we dont load it every time